_assignments/mp1.md

```diff
@@ -0,0 +1,395 @@
+```
+<!--
+![Ball Drop Script](/img/assignments/mp1/image11.png)
+-->
+
+The `OnTriggerEnter` function will be called when the collider attached to our empty GameObject is entered. The `other` parameter is the collider that intersected this collider.
+The `public GameObject` tag shows a neat feature of Unity.
+Save your script, then navigate to your empty trigger GameObject. In the inspector, the script component should look like this:
+
+![Script in Inspector](/img/assignments/mp1/image8.png)
+
+So our public GameObject is now a field for the script component in Unity. Drag the sphere into this field. Now, whenever you reference the `ball` variable in your script, it will be referencing the sphere you dragged in. Pretty neat! You can read more about this in the [Variables and the Inspector tutorial](https://docs.unity3d.com/Manual/VariablesAndTheInspector.html).
+
+Now, all that’s left is to make that sphere fall. You’ll need to get the rigid body of the sphere (rigid bodies deal with physics, read more here), using the `ball.GetComponent<Rigidbody>()` method. After that, simply set `rigidBody.useGravity` to `true`.
+
+### Break Out
+
+> Set up a button on the controllers to switch the player's position between an external viewing point and the room.
+
+Keyboards do not work well in VR, since the VR environment means that you cannot see the keyboard, and all of the keys largely feel the same. Controllers, with their contours and designated button shapes, are much easier to use blind. Using a controller in Unity is somewhat similar to the keyboard. Refer to the following pages for more information:
+
+- [Unity Input](https://docs.unity3d.com/Manual/ConventionalGameInput.html)
+- [Unity wiki page on Xbox controllers](http://wiki.unity3d.com/index.php?title=Xbox360Controller)
+- [OVR Input](https://developer.oculus.com/documentation/unity/latest/concepts/unity-ovrinput) (for Touch controllers)
+
+To move the player around the room, set the player’s `transform.position` to a `Vector3` corresponding to an empty spot in the room.
+Remember that the coordinates for the player refer to its center, and the player is 2 units tall. Thus, the player controller needs to be 1 unit above the ground.
+
+Make a script that switches the player between a room and a new external viewing point, e.g. a small new plane a moderate distance from the room. Pressing the button for the first time must move the player to the external viewing point. After that, pressing the button must alternate between the two locations.
+
+### Quit Key
+
+> Set up a button on the touch controllers to exit the game.
+
+You can add this as a separate script, or attach it in the same place as your light script. You will want to add the following lines to the `Update` method. Replace the `Input.GetKeyDown` bit with code for the controllers:
+
+![Exit game](/img/assignments/mp1/image19.png)
+
+`Application.Quit()` quits a Unity application, but it will not stop a game running in the editor.

```

Comment:
might want to also explain what the `#if` block is doing, maybe with a [link to the `UNITY_EDITOR`](https://docs.unity3d.com/Manual/PlatformDependentCompilation.html) constant definition and/or an explanation of [VS snippets in C#](https://docs.microsoft.com/en-us/visualstudio/ide/visual-csharp-code-snippets?view=vs-2017)